src/popup/router/components/AccountCard.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div
+    :class="['account-card', { subaccount: idx !== 0, 'first-subaccount': idx === 1 }]"
+    :style="cssVar"
+  >
+    <AccountInfo v-bind="$attrs" />
+    <BalanceInfo v-bind="$attrs" />
+    <div class="arrows">
+      <button @click="$emit('left')"><Arrow v-if="left" /></button>
+      <button @click="$emit('right')"><Arrow v-if="right" /></button>
+    </div>
+    <Triangle class="triangle" />
+  </div>
+</template>
+
+<script>
+import AccountInfo from './AccountInfo';
+import BalanceInfo from './BalanceInfo';
+import Triangle from '../../../icons/account-card/card-bg-triangle.svg?vue-component';
+import Arrow from '../../../icons/account-card/arrow.svg?vue-component';
+
+export default {
+  components: {
+    AccountInfo,
+    BalanceInfo,
+    Triangle,
+    Arrow,
+  },
+  props: {
+    idx: { type: Number, required: true },
+    color: { type: String, required: true },
+    shift: { type: Number, required: true },
+    left: Boolean,
+    right: Boolean,
+  },
+  computed: {
+    cssVar() {
+      return {
+        '--shift': this.shift,
+        '--color': this.color,
+      };
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../../styles/variables';
+
+.account-card {
+  display: flex;
+  flex-direction: column;
+  height: 169px;
+  width: 312px;
+  border-radius: 6px;
+  // stylelint-disable-next-line declaration-colon-newline-after
+  background: url('../../../icons/account-card/account-bg-pattern.svg'),
+    linear-gradient(rgb(10, 14, 22), rgb(10, 14, 22));
+  background-position: calc(var(--shift) * 15px) calc(var(--shift) * 20px), center, center, center;
+
+  &.subaccount {
+    // stylelint-disable-next-line declaration-colon-newline-after
+    background: url('../../../icons/account-card/subaccount-bg-pattern.svg'),
+      linear-gradient(#131313, #131313);
+    background-position: calc(var(--shift) * 15px) calc(var(--shift) * 20px), center, center, center;
+  }
+
+  &.first-subaccount.subaccount {
+    // stylelint-disable-next-line declaration-colon-newline-after
+    background: url('../../../icons/account-card/subaccount-bg-pattern.svg'),
+      linear-gradient(rgb(19, 27, 42), rgb(19, 27, 42));
```

Comment:
```suggestion
    background:
      url('../../../icons/account-card/subaccount-bg-pattern.svg'),
      linear-gradient(rgb(19, 27, 42), rgb(19, 27, 42));
```
https://github.com/stylelint/stylelint-config-standard/blob/bc51daa35a19e5cdb32188a22a07e94bb02be496/index.js#L47
https://stylelint.io/user-guide/rules/declaration-colon-newline-after#always-multi-line

Reply:
prettier and stylelint are conflicting in this place, that is the reason i used this disabling comment